demo/vl-map-wms-layer.html

```diff
@@ -39,6 +39,38 @@
         </vl-map-image-wms-layer>
       </vl-map>
     </vl-demo>
+
+    <vl-demo data-vl-title="Kaart met Image WMS kaartlaag met custom SLD">
+      <vl-map id="map-with-image-wms-layer">
+        <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
+        <vl-map-image-wms-layer
+          data-vl-name="Overstromingsgevaarkaarten"
+          data-vl-version="1.1.0"
+          data-vl-opacity="0.6"
+          data-vl-url="https://geoservice.waterinfo.be/Overstromingsgevaarkaarten-PLUVIAAL/wms"
+          data-vl-layers="Overstromingsgevaarkaarten-PLUVIAAL:overstroombaar_gebied_PLU_noCC"
+          data-vl-sld-body='<StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0.0" xsi:schemaLocation="http://www.opengis.net/sld StyledLayerDescriptor.xsd">
```

Comment:
k weet nie of ik gigantisch fan ben van "geef ne keer nen brok XML door als attribuut van een HTML element" ...

Reply:
anderzijds doen we dat ook al met JSON zegt ... enige mening van UIG man @coemans ?

Reply:
Ik had ook eens bekeken om het als slot mee te geven, maar dat was niet zo een goed idee. Wat wel een optie is, is om het wat te gaan ontleden en klassen/eigen webcomponenten van maken. Maar dit zou wellicht wat ver gaan.

Reply:
dan laat ik de keuze aan @coemans of dat OK is of niet om zo een brok data als attribuut mee te geven.

Reply:
> Ik had ook eens bekeken om het als slot mee te geven, maar dat was niet zo een goed idee.

@Pieter-Beckers kan je verduidelijken waarom het geen goed idee is?

Reply:
Die StyledLayerDescriptor XML is met casing, en als ik innerHTML of outerHTML opvroeg, dan paste die dat aan naar lowercase tags, dus niet echt wat ik wou. Misschien bestaat daar een betere oplossing voor om dat op te vragen dan?

Reply:
misschien als je met CDATA sectie doet wel ...

Reply:
Eens geprobeerd nog met de lowercase door te sturen als de CDATA, maar beiden kreeg ik niet werkend helaas. Dus als nog iemand ideeën heeft, zeker welkom, en anders zo laten als attribuut?

Reply:
Eens geprobeerd nog met de lowercase door te sturen als de CDATA, maar beiden kreeg ik niet werkend helaas. Dus als nog iemand ideeën heeft, zeker welkom, en anders zo laten als attribuut?

Reply:
Ik zie ook niet meteen een elegante oplossing om de hoofdletters te behouden via de DOM. We zouden intern een mapper kunnen schrijven, maar dat lijkt me in deze wat overkill.

Is het een optie om het attribuut te vervangen door een setter op component niveau? In dat geval moet er geen XML in een attribuut gestoken worden.

Reply:
Een setter zou kunnen werken, maar dan zitten we met het probleem dat eerst de component zou moeten gemaakt worden, de setter opgeroepen, en dan pas aan de map toegevoegd worden. Nu gebeurt het eerste en laatste eigenlijk automagisch als de layer als child aan de map wordt gehangen. Hier zie ik niet direct een propere oplossing voor.